script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -1256,6 +1257,7 @@ His manner suggests that he no longer has any interest in me or my fate.
 [sha:986d20bee85ed5ac1ea6e756829372ed0eef9cd4]{
 -- Page 747, Offset 26304.
 -- 貴方も貴方よ!?　さっき噛まれかけたクセに、まだあの怪物の本性が分からないの!?」
+// should -kun be added here since that's how Noel calls Shiki?
```

Comment:
She doesn't actually say his name here, we reified it. I guess we could add the suffix?